docker-compose.yaml

```diff
@@ -16,3 +16,5 @@ services:
     build: 
       context: .
       dockerfile: Dockerfile.dev
+    ports: 
+      - 8080:80
```

Comment:
maybe `8080` can be removed, is this not necessary to provide the port `65520` here?

Reply:
This basically forwards Remix IDE from port 80 inside of the container to port 8080 to localhost, usually, port 80 is busy and it's not recommended to expose these ports.

Sure I can add 65520 for remixd.

Reply:
Added both 8080 and 65520.